apps/condo/domains/notification/schema/SetMessageStatusService.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Generated by `createservice notification.SetMessageStatusService --type mutations`
+ */
+const get = require('lodash/get')
+
+const { GQLCustomSchema, getByCondition } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/notification/access/SetMessageStatusService')
+const { Message: MessageAPI } = require('@condo/domains/notification/utils/serverSchema')
+const {
+    MESSAGE_DELIVERED_STATUS,
+    MESSAGE_READ_STATUS,
+    MESSAGE_SENT_STATUS,
+} = require('@condo/domains/notification/constants/constants')
+
+const READ_AT_FIELD_NAME = 'readAt'
+const DELIVERED_AT_FIELD_NAME = 'deliveredAt'
+
+const MESSAGE_STATUSES = {
+    [READ_AT_FIELD_NAME]: MESSAGE_READ_STATUS,
+    [DELIVERED_AT_FIELD_NAME]: MESSAGE_DELIVERED_STATUS,
+}
+
+const ALLOWED_STATUSES_FOR_READ_AT = {
+    [MESSAGE_SENT_STATUS]: true,
+    [MESSAGE_DELIVERED_STATUS]: true,
+}
+
+const SetMessageStatusService = new GQLCustomSchema('SetMessageStatusService', {
+    types: [
+        {
+            access: true,
+            type: 'input SetMessageStatusInput { dv: Int!, sender: JSON!, messageId: String!, deliveredAt: String, readAt: String }',
```

Comment:
sender has a type. Try to avoid using JSON

Reply:
fixed